x-pack/packages/security-solution/features/src/security/kibana_sub_features.ts

```diff
@@ -554,6 +554,47 @@ const executeActionSubFeature: SubFeatureConfig = {
   ],
 };
 
+// 8.15 feature
+const scanActionSubFeature: SubFeatureConfig = {
+  requireAllSpaces: true,
+  privilegesTooltip: i18n.translate(
+    'securitySolutionPackages.features.featureRegistry.subFeatures.scanOperations.privilegesTooltip',
+    {
+      defaultMessage: 'All Spaces is required for Scan Operations access.',
+    }
+  ),
+  name: i18n.translate(
+    'securitySolutionPackages.features.featureRegistry.subFeatures.scanOperations',
+    {
+      defaultMessage: 'Scan Operations',
+    }
+  ),
+  description: i18n.translate(
+    'securitySolutionPackages.features.featureRegistry.subFeatures.scanOperations.description',
+    {
+      defaultMessage: 'Perform folder scan response actions in the response console.',
```

Comment:
This info text here may need an update. I just went with a message similar to execute action @caitlinbetz